Sources/Compiler/IR/InstructionID.swift

```diff
@@ -1,30 +1,30 @@
-/// A stable identity denoting an instruction in a module.
+/// The stable identity of an instruction in its module.
 ///
 /// - SeeAlso: `InstructionIndex`
 public struct InstructionID: Hashable {
 
-  /// The index of the function in which the instruction resides.
+  /// The function containing this identity.
   public let function: Module.Functions.Index
 
-  /// The address of the block in which the instruction resides.
+  /// The block containing this identity.
   public let block: Function.Blocks.Address
 
-  /// The address of the instruction.
+  /// The identity of the instruction in its block.
   public let address: Block.Instructions.Address
 
-  /// Creates a path with the given properties.
+  /// Creates an identity with the given properties.
```

Comment:
```suggestion
  /// Creates an instance with the given properties.
```
“creates an identity” makes me think it's doing more than just creating an instance of this type, and I wonder what else is going on.  When I see "Creates an instance” I am able to skip over it as the standard phrase for “makes one of these.”  Does that make sense?